models/model.js

```diff
@@ -42,7 +43,17 @@ exports.selectReviewByID = (ID) => {
 
     .then(({ rows }) => {
       if (rows.length !== 0) return rows[0];
+      else return Promise.reject({ status: 404, message: "Path not found" });
```

Comment:
A minor thing, but may be a useful pattern to follow. It is generally best to deal with the error first, then continue if no error.